export_kakuma_advert_contacts.py

```diff
@@ -0,0 +1,173 @@
+import argparse
+import csv
+import json
+from collections import OrderedDict
+
+from core_data_modules.logging import Logger
+from core_data_modules.cleaners import Codes
+from core_data_modules.traced_data.io import TracedDataJsonIO
+from id_infrastructure.firestore_uuid_table import FirestoreUuidTable
+from storage.google_cloud import google_cloud_utils
+
+from src.lib import PipelineConfiguration
+from src.lib.code_schemes import CodeSchemes
+
+Logger.set_project_name("WUSC-KEEP-II")
+log = Logger(__name__)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Generates a list of phone numbers and textit language of "
+                                                 "WUSC-KEEP-II-KAKUMA participants"
+                                                 "based on their manually labeled demographic language response")
+
+    parser.add_argument("google_cloud_credentials_file_path", metavar="google-cloud-credentials-file-path",
+                        help="Path to a Google Cloud service account credentials file to use to access the "
+                             "credentials bucket")
+    parser.add_argument("pipeline_configuration_file_path", metavar="pipeline-configuration-file",
+                        help="Path to WUSC-KEEP-II-KAKUMA pipeline configuration json file")
+    parser.add_argument("messages_json_input_path", metavar="messages_json_input_path",
+                        help="Path to the WUSC-KEEP-II-KAKUMA messages traced data JSONL file to extract phone"
+                             "numbers from")
+    parser.add_argument("contacts_csv_path", metavar="contacts-csv-path",
+                        help="CSV file path to write per-language contacts data to")
```

Comment:
```suggestion
                        help="CSV file path to write the contacts data to")
```
(as you're now only exporting 1 csv)

Reply:
updated thanks!